src/App.tsx

```diff
@@ -1,23 +1,92 @@
 import "./App.css";
-
+import React, { useState } from 'react';
 import { taskList } from "./taskList";
 
+let taskListCopy = taskList;
+
+// Create a list with additional field to mark deleted task
+  let taskListExt: any[] = [];
+  for (let i=0 ; i < taskListCopy.length; i++){
+    taskListExt.push(
+      {
+        'id'  : taskListCopy[i].id,
+        'name': taskListCopy[i].name,
+        'del' : '' 
+      }
+    )
+  }
+
+
+//
+
 function TaskShow() {
+  let [tasks, setTasks ] = useState(taskListExt);
+  let [newTask, setNewTask] = useState('');
+  
+  const handleSubmit:React.FormEventHandler<HTMLFormElement> = (event) => {
+    
+    event.preventDefault();
+    let newId  = tasks[ tasks.length-1 ].id + 1;
+    
+      setTasks( (previousTask) => [
+                  ...previousTask,
+                  { id: newId,
+                    name: newTask
+                  }
+              ]
+            )
+  }
+
   return (
+
     <div className="App">
+
       <h2>My Fancy TODO App</h2>
-      {taskList.map((task) => (
-        <div className="container">
-            <div className="id-container">
+      {tasks.map((task) => (
+        <div className="container" key={task.id}>
+            <div className="id-container" >
               <span className="text">{task.id}</span>
             </div>
             <div className="task-name-container">
               <span className="text">{task.name}</span>
             </div>
+
+            <div className="delete-button">
+              <button type='button'
+                onClick={ () => {
+                  task.del = '*'
+                  setTasks(tasks.filter( item => item.del !== '*'))
+                }}  
+              >
+               <font size="+3" color='red'>🗑</font>
+   
+              </button>
+            </div>  
         </div>
       ))}
+
+      <div className="new-task-container"> 
+        <form onSubmit={handleSubmit} >
+          <div>
+            <label>Task name
+              <div className="newtask-name">
+                <input 
+                    style={{width: "640px"}}
```

Comment:
Since we already do the styling in `.css`, we should not use the inline style here. I suggest putting a class name and style it outside. Also, don't forget to use `padding` for this `input` like how it is in the design too